examples/location_example_uc3_destination_charger_not_published.json

```diff
@@ -16,16 +16,18 @@
     "uid": "fd855359-bc81-47bb-bb89-849ae3dac89e",
     "evse_id": "NL*ALF*E000000001",
     "status": "AVAILABLE",
-    "connectors": [{
-      "id": "1",
-      "standard": "IEC_62196_T2",
-      "format": "SOCKET",
-      "power_type": "AC_3_PHASE",
-      "voltage": 220,
-      "amperage": 16,
-      "last_updated": "2019-07-01T12:12:11Z"
-    }],
-    "parking_restrictions": "CUSTOMERS",
+    "connectors": [

```

Comment:
Same as the other examples, this only changes the layout.

I do agree with the "parking_restrictions", that is a good find!!

Reply:
No it's also amperage to max_amperage and voltage to max_voltage.